src/ui/macstr_ui.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import yaml
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+
+class Ui_MainWindow(object):
+    def setupUi(self, MainWindow):
+
+        # loading in the JSON data
+        with open("../../notebooks/test_image_coords.json", "r") as contents:
+            image_data = yaml.safe_load(contents)
+            print(image_data["cell_image_1.tiff"])
+
+        # creating main GUI window
+        MainWindow.setObjectName("MainWindow")
+        MainWindow.resize(800, 600)
+
+        self.centralwidget = QtWidgets.QWidget(MainWindow)
+        self.centralwidget.setObjectName("centralwidget")
+
+        # ------------------------------
+        # setting up image canvas
+        # ------------------------------
+        self.image1 = QtWidgets.QLabel(self.centralwidget)
```

Comment:
Would it make sense to rename these `image#` widgets to include `label` or `lb` to help differentiate from `QtImage` in terms of readability? Some of the [examples from PyQt's docs](https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QLabel.html#detailed-description) show this style.

Reply:
I am not quite understanding what you mean about this. Is it because the `QtWidgets.QLabel(self.centralwidget)` all have the same label for each `self.image#`?